app/locales/en.json

```diff
@@ -6,11 +6,19 @@
     "authorities_desc": "Choose trusted healthcare authorities in your area to obtain exposure data. Either select a name from the global registry, or enter the web address provided by an authority which has implemented Safe Paths.",
     "authorities_input_placeholder": "Paste your URL here",
     "authorities_no_sources": "No data source yet",
+    "authorities_new_in_area_title": "New Healthcare Authority In Your Region",
```

Comment:
Take screenhots of these new strings so we can upload to lokalise

Reply:
Where should we be placing these screenshots? This is used in a push notification - so we want a screenshot of that notification to upload to Lokalise?